Ryujinx.Graphics/Graphics3d/NvGpuEngine3d.cs

```diff
@@ -407,6 +408,36 @@ private void SetStencil(GalPipelineState State)
             }
         }
 
+        private void SetScissor(GalPipelineState State)
+        {
+            int Height = ReadRegister(NvGpuEngine3dReg.FrameBufferNHeight);
+            State.ScissorTestCount = 1; // Stubbed, only the first scissor test is valid without a geometry shader loaded. At time of writing geometry shaders are also stubbed.
```

Comment:
I think you should put a "TODO" or a "FIXME" here.
```suggestion
            State.ScissorTestCount = 1; // FIXME: Stubbed, only the first scissor test is valid without a geometry shader loaded. At time of writing geometry shaders are also stubbed.
```